src/main/java/fr/inria/coming/changeminer/analyzer/instancedetector/PatternInstancesFromRevision.java

```diff
@@ -35,15 +35,15 @@ public void setInstances(List<PatternInstancesFromDiff> instances) {
 		this.infoPerDiff = instances;
 	}
 
-	@Override
-	public String toString() {
-		// return "PatternInstancesFromRevision [instances=" + instances + "]";
-		String r = "";
-		for (PatternInstancesFromDiff patternInstancesFromDiff : infoPerDiff) {
-			r += "\n" + patternInstancesFromDiff.toString();
-		}
-
-		return r;
-	}
+//	@Override
```

Comment:
undo this. a toString remains as is, we must remove the call to tostring instead.